.github/workflows/ci.yml

```diff
@@ -126,8 +126,17 @@ jobs:
       - name: Typecheck
         run: yarn run typecheck
 
+      - name: Tests (old; mocha)
+        run:  yarn workspace open-discussions run test
+        env:
+          NODE_ENV: test
+
+      - name: Get number of CPU cores
+        id: cpu-cores
+        uses: SimenB/github-actions-cpu-cores@v1
+
       - name: Tests
-        run: yarn run test
+        run: yarn test --max-workers ${{ steps.cpu-cores.outputs.count }}
```

Comment:
As recommended in https://jestjs.io/docs/troubleshooting#tests-are-extremely-slow-on-docker-andor-continuous-integration-ci-server